packages/@o3r/core/package.json

```diff
@@ -139,7 +139,7 @@
     "@ngrx/effects": "~16.2.0",
     "@ngrx/store-devtools": "~16.2.0",
     "@types/jest": "~29.5.2",
-    "nx": "~16.6.0",
+    "nx": "~16.7.0",
```

Comment:
is is a miss from renovate?
Is it strange you do not have yarnlock associated to the change